test/test-cases/scale/saic/sai_dpu_client_server_ptf.json

```diff
@@ -0,0 +1,41 @@
+{"DPU": [
+  {
+    "alias": "dash",
+    "asic": "generic",
+    "target": "bmv2",
+    "type": null,
+    "sku": null,
+    "mode": "client-server",
+    "sai_server_ip": "127.0.0.1",
+    "client": {
+        "type": "thrift",
+        "config": {
+            "ip": "127.0.0.1",
+            "port": "9092"
+        }
+    },
+    "port_groups": [{"1x10G": "Ethernet0", "init": "1x10G", "alias": 0},
+                    {"1x10G": "Ethernet1", "init": "1x10G", "alias": 1}
```

Comment:
in this port_groups, it is 1x10G, below, it is 10G, they seems to be the same, can we make them unified, e.g. 10G.

Reply:
@aputriax could you please respond?